Backoffice/NavigationPanel/Strategies/AbstractNavigationPanelStrategy.php

```diff
@@ -23,17 +24,25 @@
     protected $role = null;
 
     /**
-     * @param string $name
-     * @param string $role
-     * @param int    $weight
-     * @param string $parent
+     * @param string                   $name
+     * @param string                   $role
+     * @param int                      $weight
+     * @param string                   $parent
+     * @param array                    $datatableParameter
+     * @param TranslatorInterface|null $translator
      */
-    public function __construct($name, $role, $weight, $parent)
+    public function __construct($name, $role, $weight, $parent, $datatableParameter, $translator)
     {
         $this->name = $name;
         $this->role = $role;
         $this->weight = $weight;
         $this->parent = $parent;
+
+        if (!is_null($translator)) {
```

Comment:
You're right and that's what I did : $datatableParameter = array(), $translator = null. But whith theses default values, I was unable to see which NavigationPanelStrategy was updated with the new logic. So I decided to remove default values to obtain Exception and update strategies. I found it convenient so let it in place. Are you sure, I should add the default values? (ok for the test) @nicolasThal